test/functional/api/cas/installer.py

```diff
@@ -23,21 +24,21 @@ def install_opencas():
         "./configure && "
         "make -j")
     if output.exit_code != 0:
-        TestRun.exception(
+        raise CmdException(
             f"Make command executed with nonzero status: {output.stdout}\n{output.stderr}")
```

Comment:
pass output as second arg to CmdException and remove stdout and stderr from f-string

Reply:
Done.